src/NServiceBus.Core/Configure.cs

```diff
@@ -126,6 +127,23 @@ internal void Initialize()
             }
         }
 
+        void ConfigureBehaviors()
```

Comment:
> Just renaming 5.2.15 to 5.3.0 is not enough to support backward compatibility.

I'd argue this is not a critical bug since it doesn't really cause a significant damage. Therefore no reason to backport it.

we don't state/guarantee anything specific to `IBehavior` registration. Again, there isn't really much wrong with the current approach. It's not beautiful as structuremaps points those unnecessary registrations out, but it doesn't change the behavior at all.